datamodels/2.x/itop-portal-base/portal/src/Controller/ManageBrickController.php

```diff
@@ -70,10 +72,15 @@
  */
 class ManageBrickController extends BrickController
 {
-	/** @var string EXCEL_EXPORT_TEMPLATE_PATH 
-	 * @deprecated since 3.2.1
-	 */
-	const EXCEL_EXPORT_TEMPLATE_PATH = 'itop-portal-base/portal/templates/bricks/manage/popup-export-excel.html.twig';
+
```

Comment:
```suggestion
	/** 
	 * @var string EXCEL_EXPORT_TEMPLATE_PATH 
	 * @deprecated since 3.2.1
	 */
	const EXCEL_EXPORT_TEMPLATE_PATH = 'itop-portal-base/portal/templates/bricks/manage/popup-export-excel.html.twig';
	
```
Only remove these in 3.3.0